docs/stencil-docs/customizing-checkout/checkout-sdk-example.md

```diff
@@ -116,421 +116,24 @@ lineNumbers: true
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```shell
-# make sure you're in your theme's dir
-cd ~/path/to/theme/dir
-
-# clone the SDK example outside the theme's dir
-git clone git@github.com:bigcommerce/checkout-sdk-js-example.git ../checkout-sdk-example
-
-# add directory to hold example code
-mkdir assets/js/checkout-app
-
-# copy example source code to directory you just created
-cp -R ../checkout-sdk-example/src/* assets/js/checkout-app/
-```
-
-### Import Dependencies
-
-In `/assets/js/app.js`, add the following lines below the existing `import` statements:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```js
-import React from 'react';
-import ReactDOM from 'react-dom';
-import Checkout from './checkout-app/Checkout/checkout';
-```
-
-### Define a Function to Initialize React
-
-In `/assets/js/app.js`, add the following lines to the end:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```js
-window.initReact = function initReact() {
-    ReactDOM.render(
-        React.createElement(Checkout, null, null),
-        document.getElementById('checkout-app')
-    );
-};
-```
-
-### Customize the Checkout Template
-
-In `/templates/pages/checkout.html`, add the following code between `{{{ footer.scripts }}}` and `{{/partial}}`:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```html
-<script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
-
-<script type="text/javascript" defer>
-    // Exported in app.js
-    window.initReact();
-</script>
-```
-
-Then, in `/templates/pages/checkout.html`, replace:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```html
-{{ checkout.checkout_content }}
-```
-
-with:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```
-<div id="checkout-app"></div>
-```
+1. Navigate to the [Open source checkout](https://github.com/bigcommerce/checkout-js)repository. 

```

Comment:
I think we might want to keep the instructions of the checkout sdk example tied to the Checkout SDK Example repository. Will investigate what's required to do so.